Library/Formula/geoserver.rb

```diff
@@ -2,8 +2,9 @@
 
 class Geoserver < Formula
   homepage 'http://geoserver.org/'
-  url 'https://downloads.sourceforge.net/project/geoserver/GeoServer/2.7.0/geoserver-2.7.0-bin.zip'
-  sha1 '04184225bd8e640642dbd275b1a87a14fdb89a8e'
+  version = '2.7.1'
```

Comment:
Please revert this to how it was. We don't permit variables in the URL like this.